recipes/qarrayrun/meta.yaml

```
@@ -0,0 +1,44 @@
+{% set name = "qarrayrun" %}
+{% set version = "1.0.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "80441f5a0cf27e3ba8805ced549f27c176f43d356e6d783c33ef643585a106c5"
+
+build:
+  number: 0
+  entry_points:
+    - qarrayrun = qarrayrun.cli:main
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
```

```suggestion
  script: "{{ PYTHON }} -m pip install . -vv"
```

The code on ``conda-build`` was updated. However, I think there is not a new release with those modifications

---

Done